src/js/components/Calendar/Calendar.js

```diff
@@ -124,6 +124,8 @@ const CalendarDay = ({
   <StyledDayContainer sizeProp={size} fillContainer={fill}>
     <CalendarDayButton fill={fill} {...buttonProps}>
       <StyledDay
+        tabIndex={0}
+        aria-label={buttonProps.a11yTitle}
```

Comment:
We should make sure that if the user is trying to pass the common `aria-label` prop it will propagate as well.

Reply:
agreed with this comment, unless the `aria-label` will already be placed naturally on line 125? with {...buttonProps}

Reply:
The other question is how will a screenreader deal with a11yTitle being on both the CalendarDayButton and the StyledDay? Will it read it twice in some cases?

Reply:
i've removed this from the StyledDay as i think taylor is right about {...buttonProps}; here is a screenreader demo with that approach: 

https://user-images.githubusercontent.com/38971752/124791561-294d2f00-df1a-11eb-8dc7-68458c7ad3a6.mov